python/paddle/fluid/regularizer.py

```diff
@@ -237,6 +237,7 @@ def __call__(self, param, grad, block):
                         'Ids': idx},
                 outputs={'Out': decay},
                 attrs={'is_sparse': True})
+            param = decay
```

Comment:
After change, the 
```python
block.append_op(
             type='sign', inputs={"X": param}, outputs={"Out": decay})	             type='sign', inputs={"X": param}, outputs={"Out": decay})
```
in and out will be the same, is this for purpose?

Reply:
Yes